Azeem/src/tex2Wiki.py

```diff
@@ -312,8 +313,12 @@ def readin(ofname,glossary,mmd):
         refEqs = []
         parse = False
         head = False
+        chapRef = [("GA", open("../../data/GA.tex", 'r')), ("ZE", open(ofname, 'r'))]
```

Comment:
what ofname="../../data/GA.tex" ?
I guess there should be a more systematic approach to pass the chapRefs